README.md

```diff
@@ -1,25 +1,31 @@
 
-# EC-action-verifier
+# Enterprise Contract GitHub Actions
 
-EC to be a general purpose tool for people who want to build their software securely. It Validate conformance of container images with the Enterprise Contract.
-## Documentation
+## verifier
+"Verify" is a custom GitHub Action made by Enterprise Contract that validates container images for security and compliance. It performs a three-stage validation process:
```

Comment:
Probably should be consistent with the directory name, so choose either "verify" or "verifier" and use it consistently.